airbyte-integrations/connectors/source-amplitude/source_amplitude/streams.py

```diff
@@ -206,3 +209,11 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
 
     def path(self, **kwargs) -> str:
         return f"{self.api_version}/export"
+
+    def get_error_handler(self) -> ErrorHandler:
```

Comment:
I know it was already like that but isn't it a bit frightening that we might lose record and we just print `The file size of the exported data is too large. Shorten the time ranges and try again. The limit size is 4GB.`? @artem1205 is there a reason we don't fail with a config error asking the user to provide a different `request_time_range`?

Reply:
Good catch,
this is definitely a config error.
I think we should change this is manifest as well.

https://github.com/airbytehq/airbyte/blob/8d905e3964851a8f62e0d2462e683b800a71390a/airbyte-integrations/connectors/source-amplitude/source_amplitude/manifest.yaml#L19-L23

Reply:
Updated to align low-code response filters and stream's error_handler by raising config errors for 400 and 403. Continues to ignore on 404. And raises a transient for 504.